testing/trino-test-jdbc-compatibility-old-driver/bin/run_tests.sh

```diff
@@ -29,9 +29,11 @@ echo "Testing every ${version_step}. version between ${first_tested_version} and
 tested_versions=$(seq "${first_tested_version}" ${version_step} "${previous_released_version}" | grep -vx 404)
 
 if (( (previous_released_version - first_tested_version) % version_step != 0 )); then
-    tested_versions="${tested_versions} ${previous_released_version}"
+    tested_versions+=("${previous_released_version}")
 fi
 
+tested_versions+=("422")
```

Comment:
Why do we need to test compatibility for this specific version?

Reply:
We've discovered in internal testing that this version fails in this test (I will drop second commit).

Reply:
I'm not sure I understand how the first commit would fix a failure in the old driver. Can you explain?

Reply:
@nineinchnick can you explain?

Reply:
The old driver fails due to this missing dependency, so adding it to the classpath will allow the tests to run.

Reply:
I think the easiest solution would be to skip testing with this particular version 422, same as we do above with 404.

422 is the first affected (maybe not the only), but i don't think many versions are effected, otherwise we would know earlier.

Reply:
422, 423 and 424 are affected

Reply:
https://github.com/trinodb/trino/pull/19509